tools/cli/go-whisk-cli/commands/sdk.go

```diff
@@ -99,6 +106,58 @@ var sdkInstallCmd = &cobra.Command{
     },
 }
 
+// addToBash() attempts to find user's .bashrc and append the line
+// "eval \"`wsk sdk install bashauto`\"" to it unless the line already exists
+func addToBash() error {
+    var usr *user.User
+    var bash *os.File
+    var exists bool
+    var err error
+
+    bashCheck := "wsk sdk install bashauto"
+    bashCmd := "eval \"`wsk sdk install bashauto`\""
+
+    if usr, err = user.Current(); err != nil {
+        whisk.Debug(whisk.DbgError, "Couldn't find User Directory: %s\n", err)
+        return err
+    }
+    whisk.Debug(whisk.DbgInfo, "Found current user %s\n", usr.HomeDir)
+    bashLoc := fmt.Sprintf("%s/%s", usr.HomeDir, ".bashrc")
+    if bash, err = os.OpenFile(bashLoc, os.O_APPEND|os.O_WRONLY, 0600); err != nil {
+        whisk.Debug(whisk.DbgError, "Could not find .bashrc: %s\n", err)
+        return err
+    }
+    defer bash.Close()
+    if exists, err = lineExists(bashLoc, bashCheck); err != nil {
+        return err
+    }
+    if exists == false {
+        whisk.Debug(whisk.DbgInfo, "Attempting to append .bashrc\n")
+        if _, err :=  bash.WriteString(bashCmd); err != nil {
+            whisk.Debug(whisk.DbgError, "Could not append .bashrc: %s\n", err)
+            return err
+        }
+        whisk.Debug(whisk.DbgInfo, "Successfully appended .bashrc\n")
+    }
+
+    return nil
+}
+
+// lineExists(string, string) checks to see if a string is contained in a specified file
+// Returns a boolean (true meaning the line exists) and any errors that
+func lineExists(fileLoc string , text string) (bool, error) {
```

Comment:
This function might be useful in other areas - utils.go might be a better place for it.